cmd/lakectl/cmd/ingest.go

```diff
@@ -132,6 +133,7 @@ func init() {
 	ingestCmd.Flags().String("to", "", "lakeFS path to load objects into (e.g. \"lakefs://repo/branch/sub/path/\")")
 	_ = ingestCmd.MarkFlagRequired("to")
 	ingestCmd.Flags().Bool("dry-run", false, "only print the paths to be ingested")
+	ingestCmd.Flags().String("intg", "", "integration source which must be used for ingestion")
```

Comment:
```suggestion
	ingestCmd.Flags().String("integration-source", "", "integration source to use for ingestion")
```

"intg" is a big hard for me to understand.  I'd be happy for an alias or even a one-letter short flag if `integration-source` is too much to type.  But I _do_ like the descriptive flag name for a flag which is not used much, it is easier for me to read.